stackslib/src/chainstate/burn/db/sortdb.rs

```diff
@@ -3602,6 +3615,18 @@ impl SortitionDB {
             .try_into()
             .ok()
     }
+
+    /// Get the last block-commit from a given sender
+    pub fn get_last_block_commit_by_sender(
```

Comment:
Is this only used in tests? Can you move it to a test module?

Reply:
So it would seem.  Will move